File-Formats/DocIO/Open-and-save-Word-document-in-Windows-Forms.md

```diff
@@ -0,0 +1,119 @@
+control: DocIO
+documentation: UG
+---
+
+# Open and save Word document in Windows Forms
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, and edit **Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **create a Word document in Windows Forms**.
+
+## Steps to open and save Word document programmatically:
+
+Step 1: Create a new Windows Forms application project.
+
+Step 2: Install [Syncfusion.DocIO.WinForms](https://www.nuget.org/packages/Syncfusion.DocIO.WinForms/) NuGet package as a reference to your Windows Forms application from the [NuGet.org](https://www.nuget.org/).
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
+
+Step 3: Include the following namespaces in the **Form1.Designer.cs** file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
+using Syncfusion.DocIO.DLS;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 4: Add a new button in **Form1.Designer.cs** to create Word file as follows.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+private Button btnOpenAndSave;
+private Label label;
+
+private void InitializeComponent()
+{
+    label = new Label();
+    btnOpenAndSave = new Button();
+
+    //Label
+    label.Location = new System.Drawing.Point(0, 40);
+    label.Size = new System.Drawing.Size(426, 35);
+    label.Text = "Click the button to view a Word document generated by Essential DocIO. Please note that Microsoft Word Viewer or Microsoft Word is required to view the resultant Word document";
+    label.TextAlign = System.Drawing.ContentAlignment.MiddleCenter;
+
+    //Button
+    btnOpenAndSave.Location = new System.Drawing.Point(180, 110);
+    btnOpenAndSave.Size = new System.Drawing.Size(85, 36);
+    btnOpenAndSave.Text = "Open and Save Document";
+    btnOpenAndSave.Click += new EventHandler(btnOpenAndSave_Click);
+
+    //Create Word
+    ClientSize = new System.Drawing.Size(450, 150);
+    Controls.Add(label);
+    Controls.Add(btnOpenAndSave);
+    Text = "Open and Save Word";
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 5: Add the following code in **btnOpenAndSave_Click** to **Open and save Word document** with simple text.
+
+**Open an existing Word document:**
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Open an existing Word document.
+using (WordDocument document = new WordDocument(Path.GetFullPath(@"../../Data/Input.docx"), FormatType.Automatic))
+{
+
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+**Add a paragraph to the Word document:**
```

Comment:
Refer this file and change the steps and remove this line
https://github.com/syncfusion-content/fileformat-docs/blob/hotfix/hotfix-v21.1.35/File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET.md

Reply:
Done